src/Compilers/CSharp/Portable/Symbols/Attributes/WellKnownAttributeData/ParameterWellKnownAttributeData.cs

```diff
@@ -7,50 +7,98 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal sealed class ParameterWellKnownAttributeData : CommonParameterWellKnownAttributeData
     {
-        private bool _hasNotNullWhenTrueAttribute;
-        public bool HasNotNullWhenTrueAttribute
+        private bool _hasAllowNullAttribute;
```

Comment:
>_hasAllowNullAttribute [](start = 21, length = 22)

We might want to pack these bools.

Reply:
Will leave that for later.

---
In reply to: [286263442](https://github.com/dotnet/roslyn/pull/35846#discussion_r286263442) [](ancestors = 286263442)